src/itype.h

```diff
@@ -1446,6 +1446,9 @@ struct itype {
         */
         float insulation_factor = 1.0f;
 
+        /** Flat damage reduction (increase if negative) on fall (some logic may apply). */
+        float fall_damage_reduction = 0;
```

Comment:
Not used as a float.
```suggestion
        int fall_damage_reduction = 0;
```